src/AspNetCoreModuleV2/OutOfProcessRequestHandler/processmanager.cpp

```diff
@@ -109,67 +63,53 @@ PROCESS_MANAGER::GetProcess(
     _Out_   SERVER_PROCESS            **ppServerProcess
 )
 {
-    HRESULT          hr = S_OK;
-    BOOL             fSharedLock = FALSE;
-    BOOL             fExclusiveLock = FALSE;
     DWORD            dwProcessIndex = 0;
-    SERVER_PROCESS  *pSelectedServerProcess = NULL;
+    std::unique_ptr<SERVER_PROCESS>  pSelectedServerProcess;
```

Comment:
I'm iffy on this change being in this PR.

Reply:
Why?

Reply:
Resolved offline. Keeping change in PR due to throwing on null.